src/backend/templates/schedule.hbs

```diff
@@ -460,6 +460,7 @@
             trackFilterMode = 1;
             roomFilterMode=0;
             var local_storage_events=JSON.parse(localStorage[eventName]);
+            var isRoom = 1;
```

Comment:
Although this may not be the case in the other parts of the app right now, please use `true` or `false` for binary flags to make it more readable